### PR TITLE
feat: add sitemap

### LINKED ISSUE #180: [ SEO ] Sitemap absent on the website

### PR DESCRIPTION
**Pull request by ctrl07**
Closes #180

## Changes proposed

Added sitemap

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/app/sitemap.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default function sitemap() {
+  return [
+    {
+      url: 'https://eddiehub.org',
+      lastModified: new Date(),
+      changeFrequency: 'monthly',
+      priority: 1,
+    },
+  ]
+}
```
Comment (xkrishguptaa) [inline src/app/sitemap.js:10]:
```suggestion
}

```

Comment (ctrl07) [reply src/app/sitemap.js:10]: Hello Krish,

Please can you help me understand why we need a space, I am new to this your support will be greatly appreciated!

Thanks!

Comment (xkrishguptaa) [reply src/app/sitemap.js:10]: @ctrl07 We have a newline at the end in all of our files, so it is to maintain consistency

Comment (ctrl07) [reply src/app/sitemap.js:10]: Thanks! I will keep it in mind.

Comment (ctrl07) [reply src/app/sitemap.js]: I completed it from the suggestions itself. Not able to figure out if I got it correctly.

**Pull request merged**